src/main/java/com/newworld/saegil/simulator/controller/TextToSpeechController.java

```diff
@@ -0,0 +1,34 @@
+package com.newworld.saegil.simulator.controller;
+
+import org.springframework.core.io.buffer.DataBuffer;
+import org.springframework.core.io.buffer.DefaultDataBufferFactory;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.newworld.saegil.simulator.dto.TextToSpeechRequest;
+import com.newworld.saegil.simulator.service.TextToSpeechService;
+
+import reactor.core.publisher.Flux;
+
+@RestController
+@RequestMapping("/api/v1/tts")
+public class TextToSpeechController {
+
+	private final TextToSpeechService textToSpeechService;
+
+	public TextToSpeechController(TextToSpeechService textToSpeechService) {
+		this.textToSpeechService = textToSpeechService;
+	}
+
```

Comment:
아까 캐치를 못했는데 롬복의 `@RequiredArgsConstructor`를 사용하면 생성자 코드를 적지 않아도 된답니다

```suggestion
@RestController
@RequestMapping("/api/v1/tts")
@RequiredArgsConstructor
public class TextToSpeechController {

	private final TextToSpeechService textToSpeechService;

```

Reply:
우와 이거, 변경제안도 할 수 있군요? 깃허브에서 처음 안 기능...